workflow-templates/re-encrypt-sops-secrets.yaml

```
@@ -37,14 +37,14 @@ jobs:
           repo_public_key=$(cat .github/.gpg | base64 | tr -d '\n')
           GPG_KEYS+=( "$repo_public_key" )
           public_gpg_keys=$(printf '%s\n' "${GPG_KEYS[@]}" | jq -R . | jq -s . | jq -rc .)
-          echo "::set-output name=base64_encoded::$public_gpg_keys"
+          echo "base64_encoded=$public_gpg_keys" >> $GITHUB_OUTPUT
       - name: create a local branch on the repository
         id: branch
         working-directory: actions/generator/workspace
         run: |
           branch="secret-management/sops/reencryption-${{ github.run_id }}"
           git checkout -b "$branch"
-          echo "::set-output name=work_branch::$branch"
+          echo "work_branch=$work_branch" >> $GITHUB_OUTPUT
```

This seems to be a typo:

```suggestion
          echo "work_branch=$branch" >> $GITHUB_OUTPUT
```